src/main.rs

```diff
@@ -31,6 +31,8 @@ extern crate env_logger;
 extern crate foxbox_users;
 #[macro_use]
 extern crate iron;
+#[cfg(test)]
```

Comment:
Nit: I'd move this import with the [other test dependencies](https://github.com/fxbox/foxbox/pull/101/files#diff-639fbc4ef05b315af92b4d836c31b023R22). Btw, they are probably [best placed here](https://github.com/fxbox/foxbox/pull/101/files#diff-639fbc4ef05b315af92b4d836c31b023R47).